tests/validation/tests/v3_api/test_import_cluster.py

```diff
@@ -17,6 +17,8 @@
 AWS_SSH_KEY_NAME = os.environ.get("AWS_SSH_KEY_NAME")
 AWS_NODE_COUNT = int(os.environ.get("AWS_NODE_COUNT", 3))
 HOST_NAME = os.environ.get('RANCHER_HOST_NAME', "testcustom")
+RKE_K8S_VERSION = os.environ.get("RANCHER_RKECLUSTER_K8S_VERSION",
```

Comment:
Can we rename the env var to `os.environ.get("RANCHER_RKE_K8S_VERSION"`

Reply:
done